.github/workflows/nightly_teleport_operations_procedure_stable.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: Nightly Teleport EKS Operational Procedure Test Stable
```

Comment:
Same as the other comment on the workflow to rearrange the naming and remove nightly.
Just for better visualization of the new workflows.